src/components/search/Search.js

```diff
@@ -1,14 +1,78 @@
-import { SearchAlt2 } from '@styled-icons/boxicons-regular'
+import '@reach/combobox/styles.css'
+
+import {
+  Combobox,
+  ComboboxInput,
+  ComboboxList,
+  ComboboxOption,
+} from '@reach/combobox'
+import { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import usePlacesAutocomplete, {
+  getGeocode,
+  getLatLng,
+} from 'use-places-autocomplete'
 
 import Input from '../ui/Input'
+import SearchEntry from './SearchEntry'
+
+const Search = () => {
+  // const [swlat, setSwlat] = useState(0)
+  // const [swlng, setSwlng] = useState(0)
+  // const [nelat, setNelat] = useState(0)
+  // const [nelng, setNelng] = useState(0)
+  const [centerLat, setCenterLat] = useState(0)
+  const [centerLng, setCenterLng] = useState(0)
```

Comment:
Reminder that you probably won't be using any useState and even removing useState in MapPage

Reply:
Please look into using this library, actually! https://github.com/pbeshai/use-query-params#readme

Reply:
You mentioned putting an object in query params. This library seems to be a good approach.